apps/backend/src/modules/users/user.service.ts

```diff
@@ -1,11 +1,12 @@
+import { type UserResponseDto } from "@git-fit/shared/src/modules/users/libs/types/user-response-dto.type.js";
```

Comment:
Better to import it from ./libs/types/types.js

Reply:
Yeah, thanks, forgot about it) Fixed!